### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by smaharj1**
- Added Docker local development section
- Modified Readme file
- Added templates (PR template, Issue templates, code of conduct)

[NOTE]: Do not squash and merge this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,8 +76,7 @@ MongoDB, Express.js, Vue.js, Node.js, Mongoose ORM, Redis, Sendgrid, Nodemailer
 
 ## Contribution
 
-We are currently in the process of building test suites, docker images and writing extensive documentation to speed up startup time for developers. In the meantime, any Pull Requests/Issues found are appreciated.
-
+We are currently in the process of building test suites, and writing extensive documentation to speed up startup time for developers. In the meantime, any Pull Requests/Issues found are appreciated.
```
Comment (Viveckh) [inline README.md:79]: Maybe a Quickstart section here, that points to the docker setup section of the documentation will be useful here for contributors.

File: documentation/docs/dockersetup/dockersetup.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+`docker-compose` is creating multiple images and starting multiple containers each necessary for the application to run. These steps happen with `docker-compose`:
+
+### Mongo Setup
+   
+This step creates a mongo instance in a container. It exposes port `27000` if you want to access mongo through tools like `Robo3T` or CLI.
+
+### Redis Setup
+
+Redis instance is also create since the application uses session caching. The exposed port is `6379`.
```
Comment (Viveckh) [inline documentation/docs/dockersetup/dockersetup.md:40]: typo *created

**Review by Viveckh — COMMENTED** [review]
Added some minor comments for enhancement.

File: README.md (hunk 1 above)
Comment (smaharj1) [reply README.md:79]: ah. that's what you meant.

Comment (smaharj1) [reply README.md:79]: I'll add the quickstart at the top of the page.

**Review by Viveckh — APPROVED** [review]
Nice Work!

**Review by Viveckh — COMMENTED** [review]
Error occurred in deploy preview in netlify for all three UI projects. Error message attached.

image

**Pull request merged**